core/src/main/java/org/elasticsearch/rest/action/cat/RestTemplatesAction.java

```diff
@@ -85,7 +87,7 @@ private Table buildTable(RestRequest request, ClusterStateResponse clusterStateR
             if (patternString == null || Regex.simpleMatch(patternString, indexData.name())) {
                 table.startRow();
                 table.addCell(indexData.name());
-                table.addCell(indexData.getTemplate());
+                table.addCell(String.format(Locale.ROOT, "[%s]", String.join(", ", indexData.getTemplate())));
```

Comment:
I do really with it were `indexData.getTemplate*s*()` or `indexData.getPatterns()` though.